app/src/main/java/org/walleth/activities/FullscreenQRCodeActivity.kt

```diff
@@ -0,0 +1,49 @@
+package org.walleth.activities
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.view.MenuItem
+import android.view.WindowManager
+import kotlinx.android.synthetic.main.activity_fullscreen_qrcode.*
+import org.walleth.R
+import org.walleth.functions.setQRCode
+
+const val KEY_ERC681 = "extra_erc681"
+
+class FullscreenQRCodeActivity : AppCompatActivity() {
+    private var fullBrightnessSet = false
```

Comment:
this variable is never read - what is the intention here?

Reply:
It is to allow a setting as in PassDroid. I have removed it now.